social_core/utils.py

```diff
@@ -225,6 +225,8 @@ def wrapper(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except requests.HTTPError as err:
+            social_logger.error(err.response.text)
```

Comment:
Logging just the text response might be confusing, it can also be empty.

```suggestion
            social_logger.error("request failes with %d: %s", err.response.status_code, err.response.text)
```

Reply:
Yeah, better. Thanks.